moped-etl/arcgis/settings.py

```diff
@@ -2,9 +2,9 @@
 
 LAYER_IDS = {"points": 0, "lines": 1, "combined": 2}
 
-COMPONENTS_QUERY = """
-{
-  component_arcgis_online_view {
+COMPONENTS_QUERY_BY_LAST_UPDATE_DATE = """
+query GetProjectsComponents($where: component_arcgis_online_view_bool_exp!) {
```

Comment:
Using `component_arcgis_online_view_bool_exp` here lets us pass a date filter for finding records from recently updated projects or no date filter for full replace. You can see that in the `make_hasura_request` calls. Also lets us reuse the same query w/o using a library to create gql fragments or something to that effect.

Reply:
this is nice!